discourse_migration/discourse-categories.md

```diff
@@ -0,0 +1,75 @@
+# Proposed Discourse Categories
+
+* Announcements
+
+* Releases
+
+* Clang Compiler & Tools
+  - Using Clang
+  - Building Clang
+  - clangd
+
+* Subprojects
+  - lld
+  - Flang
+  - lldb
+  - Static Analyzer
+
+* Code Generation
+  - AArch64
+  - AMDGPU
+  - ARC
+  - ARM
+  - AVR
+  - BPF
+  - CSKY
+  - Hexagon
+  - Lanai
+  - M68k
+  - Mips
+  - MSP430
+  - NVPTX
+  - PowerPC
+  - RISCV
+  - Sparc
+  - SystemZ
+  - VE
+  - WebAssembly
+  - X86
+  - XCore
+
+* IR & Optimizations
+  - Loop Optimizations
+
+* Community
+  - Women in Compilers and Tools
+  - Job Postings
+  - US Developer Meeting
+  - EuroLLVM
+  - GSoC
+  - Community.o
+  - Foundation
```

Comment:
I know it is inconsistent, but I'd recommend "LLVM Foundation" since that's a proper name.